app/presenters/url.rb

```diff
@@ -0,0 +1,22 @@
+class Url
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers::UrlHelper
+
+  attr_reader :link_text, :path_name, :params
+
+  def initialize(link_text:, path_name:, params: {})
+    @link_text = link_text
+    @path_name = path_name
+    @params = params
+  end
+
+  def to_s
```

Comment:
`to_s` gets called on anything in the view. So this is a shorthand way of turning all `Url.new` instances into links when we pass them to a view template. We have this defined on `app/services/random_phrase.rb`